yarn.lock

```diff
@@ -4251,15 +4261,15 @@ __metadata:
   languageName: node
   linkType: hard
 
-"debug@npm:^4.3.5":
-  version: 4.3.5
-  resolution: "debug@npm:4.3.5"
+"debug@npm:^4.3.6":
```

Comment:
There are extraneous changes to this file, possibly due to how you changed the dependencies. There should also be a new file for `testing-library/user-event` under the `.yarn` folder, which is missing.

The easiest way to fix this is to do the following: first, revert *all* changes to `yarn.lock`, `package.json`, and `.pnp.cjs`, and `.yarn` (which you didn't commit, which leaves me thinking you may have uncommitted changes in your local repo). In other words, the dependencies should look *exactly* the same as on master, and running `git status` should not show any changes (including untracked changes) to any of them.

Then, run `yarn add -D @testing-library/user-event` and the commit all of the changes to the above files/folders I mentioned.

Reply:
I see, my bad. I generally avoided to add `.pnp` and the `.yarn` folder because these are automatically generated with `yarn install`. I followed your instructions, are they ok now?